packetbeat/protos/http/http.go

```diff
@@ -514,6 +512,15 @@ func (http *httpPlugin) newTransaction(requ, resp *message) beat.Event {
 	}
 }
 
+func (http *httpPlugin) getRaw(m *message) []byte {
```

Comment:
maybe rename this to `buildRawMessage` or `makeRawMessage`. Quite some allocation and copying is happening here.